addons/dark-www/addon.json

```diff
@@ -1670,8 +1670,6 @@
           "settingId": "button"
         },
         "options": {
-          "#4d97ff": "#28a5da",
-          "#4d97ffff": "#28a5da",
```

Comment:
Removing this breaks the "Scratch's default colors (blue)" preset:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/3607e9c2-33e1-41aa-b24f-387e5d099698)

It should look the same as the 2.0 presets:
![image](https://github.com/ScratchAddons/ScratchAddons/assets/51849865/ef19b3d1-5ddc-47cc-8289-6380c276a910)